### ISSUE TITLE
New release ?🤔 

### ISSUE THREAD

**Issue by kloczek** [acmel/dwarves#57]
https://github.com/acmel/dwarves/compare/v1.27...master shows  +90 commits since last release.
I think that it would be good to flush those commits as new release.

**Comment by alan-maguire** [comment]
yep, we're on final approach for v1.28 release and are intensively testing towards that goal. We took a bit more time with this release as there were serious memory utilization problems which needed resolving; that's done now (though there are always improvements that could be made of course). Any testing you could do on your end with the next branch would be greatly appreciated and will help contribute to the quality of the release.

**Comment by alan-maguire** [comment]
closed now since we've released v1.28
